source/kraken/worker.cpp

```diff
@@ -674,6 +674,97 @@ pbnavitia::Response Worker::pt_ref(const pbnavitia::PTRefRequest &request,
                                     current_datetime);
 }
 
+pbnavitia::Response Worker::graphical_isochron(const pbnavitia::GraphicalIsochronRequest &request_iso,
+                                               const boost::posix_time::ptime& current_datetime) {
+
+    const auto data = data_manager.get_data();
+    this->init_worker_data(data);
+
+    const auto request = request_iso.journeys_request();
+    const auto rt_level = get_realtime_level(request.realtime_level());
+
+    /// Accessibility params
+    type::AccessibiliteParams accessibilite_params;
```

Comment:
hum this seems  to dupplicate the journeys request handling.

maybe we can factorize the code no ?